vendor_skins/Evercast/deps/obs-scripting/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 2.8)
+
+if(NOT ENABLE_SCRIPTING)
+	message(STATUS "Scripting plugin disabled")
+	return()
+endif()
+
+project(obs-scripting)
+
+if(MSVC)
+	set(obs-scripting_PLATFORM_DEPS
+		w32-pthreads)
+endif()
+
+if(APPLE)
+	set(obs-scripting_PLATFORM_DEPS
+		objc)
+endif()
+
+option(DISABLE_LUA "Disable Lua scripting support" OFF)
+option(DISABLE_PYTHON "Disable Python scripting support" OFF)
+
+set(COMPILE_PYTHON FALSE CACHE BOOL "" FORCE)
+set(COMPILE_LUA FALSE CACHE BOOL "" FORCE)
+
+if(NOT DISABLE_LUA)
+	find_package(Luajit QUIET)
+
+	if(NOT DISABLE_LUA AND NOT LUAJIT_FOUND)
+		message(STATUS "Luajit support not found.")
+		set(LUAJIT_FOUND FALSE)
+	else()
+		message(STATUS "Scripting: Luajit supported")
+		set(COMPILE_LUA TRUE CACHE BOOL "" FORCE)
+	endif()
+else()
+	message(STATUS "Scripting: Luajit support disabled")
+	set(LUAJIT_FOUND FALSE)
+endif()
+
+if(NOT DISABLE_PYTHON)
+	find_package(PythonDeps QUIET)
+
+	if(NOT DISABLE_PYTHON AND NOT PYTHONLIBS_FOUND)
+		message(STATUS "Python support not found.")
+		set(PYTHON_FOUND FALSE)
+		set(PYTHONLIBS_FOUND FALSE)
+	else()
+		message(STATUS "Scripting: Python 3 supported")
+		set(PYTHON_FOUND TRUE)
+		set(COMPILE_PYTHON TRUE CACHE BOOL "" FORCE)
+
+		get_filename_component(PYTHON_LIB "${PYTHON_LIBRARIES}" NAME)
+		string(REGEX REPLACE "\\.[^.]*$" "" PYTHON_LIB ${PYTHON_LIB})
+
+		if(WIN32)
+			string(REGEX REPLACE "_d" "" PYTHON_LIB "${PYTHON_LIB}")
+		endif()
+	endif()
+else()
+	message(STATUS "Scripting: Python 3 support disabled")
+	set(PYTHON_FOUND FALSE)
+	set(PYTHONLIBS_FOUND FALSE)
+endif()
+
+find_package(SwigDeps QUIET 2)
+
+if(NOT SWIG_FOUND)
```

Comment:
SWIG is used to provide an interface layer between native (C/CPP) and other code; in fact we can probably remove it from the dependencies, although I will leave it for now, as it may be more involved.